fenix/app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -188,6 +188,7 @@ class AwesomeBarView(
                 title = R.string.search_engine_suggestions_title,
                 description = activity.getString(R.string.search_engine_suggestions_description),
                 searchIcon = searchWithBitmap,
+                charactersThreshold = SEARCH_ENGINE_SUGGESTION_PROVIDER_CHARACTERS_THRESHOLD,
```

Comment:
I think we should bump `DEFAULT_CHARACTERS_THRESHOLD` to 2 as the default since it is pretty sensible that we don't want to display a search engine suggestion because a user typed "a".